bin/read-google-sheets-moc-v2.js

```diff
@@ -0,0 +1,74 @@
+const google = require('googleapis');
+const keys = require('../creds-google-sheet.json');
+const googleMethods = require('../server/recess-events/google-methods');
+const firebasedb = require('../server/lib/setupFirebase.js').realtimedb;
+
+const SHEETS_ID = process.env.SHEETS_ID_MOC_EB_IDS;
+const SCOPES = ['https://www.googleapis.com/auth/spreadsheets'];
+
+// TODO: change this to actual path
+const ROOT_PATH_MOC = 'test_moc';
+
+// TODO: move this into models script
+class MOC {
+    constructor({ id, name, district, is_incumbent, eventbrite_ids, campaign_eventbrite_ids, is_congress_115, is_congress_116 }) {
+        this.id = id;
+        [this.first_name, this.last_name] = name.trim().split(' ', 2);
+        this.district = district;
+        this.is_incumbent = is_incumbent ? (is_incumbent.toLowerCase() == 'true') : false;
+        this.eventbrite_ids = (eventbrite_ids || '').split(',').map(id => id.trim());
+        this.campaign_eventbrite_ids = (campaign_eventbrite_ids || '').split(',').map(id => id.trim());
+        this.is_congress_115 = is_congress_115 ? (is_congress_115.toLowerCase() == 'true') : false;
+        this.is_congress_116 = is_congress_116 ? (is_congress_116.toLowerCase() == 'true') : false;
```

Comment:
What about members of the current, 117'th, Congress? And will we need to update this code when we get to the 118'th Congress? I think it's fine if we will, but maybe we should include some short instructions on the changes that will need to be made?